packages/nuka/cypress/integration/infinity-carousel.spec.js

```diff
@@ -22,16 +26,13 @@ context('Infinity Carousel', () => {
       cy.get('button').contains('Prev').should('not.be.disabled');
       cy.get('button').contains('Next').should('not.be.disabled').click();
 
-      cy.wait(500);
```

Comment:
Cypress will wait for animations to finish, so the `cy.wait`s are unnecessary:
https://docs.cypress.io/guides/core-concepts/interacting-with-elements#Animations